phpBB/includes/functions_content.php

```diff
@@ -1280,6 +1281,19 @@ function parse_attachments($forum_id, &$message, &$attachments, &$update_count_a
 					$update_count_ary[] = $attachment['attach_id'];
 				break;
 
+				// Audio files
+				case ATTACHMENT_CATEGORY_AUDIO:
+					$inline_link = append_sid("{$phpbb_root_path}download/file.$phpEx", 'id=' . $attachment['attach_id']);
+					$block_array += array(
+						'S_AUDIO_FILE'			=> true,
+						'U_FORUM'			=> generate_board_url(),
+						'ATTACH_ID'			=> $attachment['attach_id'],
+					);
```

Comment:
Short array syntax `[]` can be used here (actually this is what being done for all new core code applying arrays).